### ISSUE TITLE
collected fixes for LibPack 12.5.3

### ISSUE THREAD

**Issue by donovaly** [apeltauer/FreeCAD#19]
Here is  a zip file: https://github.com/donovaly/FreeCADInstProj/releases/download/LibPack-fixes-1/Changes-for-LibPack-12-5-3.zip

which can be used the following way:
- take the latest LibPack 12.5.3: https://github.com/apeltauer/FreeCAD/releases/download/LibPack_12.5.3/FreeCADLibs_12.5.3_x64_VC17.7z
- unzip the fixes file over it (allow to overwrite existing files on unzipping)

Changelog:
- fix 2 wrong CMake files, see #14
- replace a broken TKService.dll, see #15
- upgrade from Qt 5.15.1 to 5.15.2 (its bugfixes are needed for FreeCAD 0.19), see #17
- add missing DLLs one new for certain older Windows PCs (look at the bin folder of the ZIP file), see #18
- delivers GMSH 4.8.4 (we found out that for FEM we need GMSH 4.8 and also at the fixes in 4.8.3), see #16

---------------
I also created a ZIP file that includes the changes to the LibPack: https://github.com/donovaly/FreeCADInstProj/releases/download/LibPack-fixes-1/FreeCADLibs_12.5.3_x64_VC17_fixes_US.7z

**Comment by chennes** [comment]
Using your combined fixes to compile in debug mode I am getting the following error:
```
CMake Error at E:/FreeCAD/FreeCADLibs_12.5.3_x64_VC17_fixes_US/FreeCADLibs_12.5.3_x64_VC17/lib/cmake/Qt5Gui/Qt5GuiConfig.cmake:14 (message):
  The imported target "Qt5::Gui" references the file

     "E:/FreeCAD/FreeCADLibs_12.5.3_x64_VC17_fixes_US/FreeCADLibs_12.5.3_x64_VC17/plugins/imageformats/qgifd.dll"

  but this file does not exist.

Call Stack (most recent call first):
  lib/cmake/Qt5Gui/Qt5GuiConfig.cmake:228 (_qt5_Gui_check_file_exists)
  lib/cmake/Qt5Gui/Qt5Gui_QGifPlugin.cmake:6 (_populate_Gui_plugin_properties)
  lib/cmake/Qt5Gui/Qt5GuiConfig.cmake:237 (include)
  lib/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake:105 (find_package)
  cMake/FreeCAD_Helpers/SetupQt.cmake:15 (find_package)
  CMakeLists.txt:69 (include)
```

It appears that none of the debug versions of these image format libraries are present in the zip.

**Comment by donovaly** [comment]
Yes, my fixes ZIP only contained files necessary for releases since my focus was to finish the release of FC 0.19.3.
But since today there is a new LibPack available so my ZIP is no longer necessary.

**Comment by donovaly** [comment]
This issue can be closed once #22 is fixed.

**Comment by donovaly** [comment]
#22 still stand but this issue can be closed
